docs/examples/Themis-server/go/ssession_server.go

```diff
@@ -151,12 +181,31 @@ func main() {
 			return
 		}
 		wrapped, err := clientSession.Wrap(line)
+		if err != nil {
+			fmt.Println("Failed to encrypt message:", err)
+			return
+		}
 		data, err := sendMessage(wrapped, endpoint)
 		if err != nil {
 			fmt.Println("Error occurred:", err)
 			return
 		}
 		unwrapped, _, err := clientSession.Unwrap(data)
+		if err != nil {
+			fmt.Println("Failed to decrypt message:", err)
+			return
+		}
 		fmt.Println(string(unwrapped))
 	}
 }
+
+func main() {
+	inputBuffer := bufio.NewReader(os.Stdin)
+
+	clientSession, endpoint := createSecureSession(inputBuffer)
+	if clientSession == nil {
+		return
```

Comment:
maybe we should add `os.Exit(1)` and some message that will explain why nothing was done?

Reply:
We print an error message earlier, before returning nil here. I guess it will be enough to just exit with non-zero exit code to communicate a failure to the user: they will see the message and their terminal is likely to display a failure.

I'll add proper exit code here and in the Secure Message server example.

Reply:
@Lagovas done.

I've improved error forwarding a bit, moved the error message printing to main() and added exiting with non-zero status code. I've also done the same to Secure Message example for the sake of consistency.